virttest/test_setup.py

```diff
@@ -1313,6 +1303,23 @@ def get_pf_mlx(self):
             cmd, shell=True, timeout=60).splitlines()
         return (PF_devices)
 
+    def get_pf_ids(self):
+        """
+        Get the PF devices on x86 host
+        """
+        cmd = "lspci | grep -v 'Virtual Function' |awk '/%s/ {print $1}'" % self.pf_filter_re
```

Comment:
why not use  cmd = "lspci | awk '/%s/ {print $1}'" % self.pf_filter_re  as before ? 
IMHO, this is mush easier to understand, and the one you added use "Virtual Function" is easy to misunderstanding.